docs/source/virtual_rainforest/main_simulation.md

```diff
@@ -0,0 +1,65 @@
+# Virtual Rainforest simulation flow
+
+This document describes the main simulation flow of the Virtual Rainforest model. This
+flow will now be set out step by step.
+
+## Loading configuration files
+
+As a first step, configuration files are loaded in and validated. These files are of
+`toml` format and are found based on paths provided by the user. When these files are
+loaded in they are validated to ensure that they are valid `toml`, and that no tags are
+repeated. In cases where a required tag is not present in any file, the validation will
+fail unless a default value for it has been set. Further details can be found in the
+[configuration documentation](./core/config.md).
+
+## Grid creation
+
+The next step is creating a grid, which defines the area and resolution of the
+simulation. The [grid documentation](./core/grid.md) describes this object further. This
+grid is then used to create an empty `Data` object of the correct size to describe the
+simulation.
+
+## Loading and validation of input data
+
+This `Data` object now needs to be populated with data. The data used in the simulation
+is stored in a set of data files which are specified in the configuration. This data
+is then loaded using the `data.load_data_config` method, which has built in validation
+procedures to ensure that the loaded data can be mapped onto the grid. Further details
+can be found in the [data system](./core/data.md) and [core axes](./core/axes.md)
+documentation.
+
+## Configuration of specific modules
+
+The models used for the specific simulation run can be set in the configuration. This
+will generally be the four standard modules (`abiotic`, `animals`, `plants` and `soil`),
+but it is also useful to have the ability to run novel modules or different combinations
+of the modules. Once a list of modules to configure has been extracted from the
+configuration, they are then all configured.
```

Comment:
```suggestion
## Configuration of specific models

The Virtual Rainforest is implemented as model objects, each of which is responsible for simulating
a particular aspect of the rainforest ecosystem and the models used for a specific simulation run
can be set in the configuration. This will typically be the four standard models
(`AbioticModel`, `AnimalModel`, `PlantModel` and `SoilModel`),
but this can be extended to include new models or different combinations
of models. Once a list of models to configure has been extracted from the
configuration, they are then all configured.
```

Reply:
And maybe also link to the new model docs. "For more information about implementing new models, see this page about the required module structure and the model API."